app/views/course/experience_points_records/index.html.slim

```diff
@@ -1,3 +1,6 @@
+- add_breadcrumb t('breadcrumbs.course.users.index'), :course_users_path
```

Comment:
These should be handled by the controller. If the controllers properly inherit, this should not be needed